src/Symfony/Bundle/WebProfilerBundle/DependencyInjection/ProfilerPass.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\WebProfilerBundle\DependencyInjection;
```

Comment:
In `AddConsoleCommandPass` you were moving the file from `Symfony\Bundle\FrameworkBundle` to the component.
Here I am moving the file from `Symfony\Bundle\FrameworkBundle` to `Symfony\Bundle\WebProfilerBundle`.
After a short looking I figured that every "Bundle" in that namespace have a repository `Compiler` in there `DI`folder so I think in that case I should move it in a subfolder.